keycode.cabal

```diff
@@ -37,7 +37,7 @@ library
                      , containers
                      , ghc-prim
   if impl(ghc >= 8.0)
-    build-depends:     template-haskell >= 2.11 && < 2.13
+    build-depends:     template-haskell >= 2.11.1.0 && < 2.13
```

Comment:
OK... I don't have access to GHCJS, so I'm unable to reproduce this error you're claiming to experience. But I'm certainly not going to accept any change which excludes `template-haskell-2.11.0.0` from install plans, since that version of `template-haskell` is bundled with GHC 8.0.1.